projects/aca-folder-rules/src/lib/mock/node.mock.ts

```diff
@@ -0,0 +1,19 @@
+export const dummyNodeInfo = {
```

Comment:
We still need licensing information on every TypeScript file. I'm surprised the linter doesn't pick up on that.